src/tokens/lst_tokens/token_new.c

```diff
@@ -6,12 +6,21 @@
 /*   By: mcourtoi <mcourtoi@student.42.fr>          +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2023/03/08 19:37:40 by mcourtoi          #+#    #+#             */
-/*   Updated: 2023/03/19 19:04:00 by mcourtoi         ###   ########.fr       */
+/*   Updated: 2023/04/03 17:55:59 by mcourtoi         ###   ########.fr       */
 /*                                                                            */
 /* ************************************************************************** */
 
 #include "minishell.h"
 
+/**
+ * @brief Create a new token with the given parameter type and str.
+ * 
+ * @details Uses ft_strdup() for the str.
+ * 
+ * @param type of the new token.
+ * @param str of the new token
```

Comment:
Remember that the parameter name and its description have to be considered separated.